nextcloudappstore/core/templates/base.html

```diff
@@ -9,6 +9,7 @@
     <meta name="author" content="">
     <link rel="shortcut icon" href="{% static 'assets/img/favicon.png' %}"/>
     <title>Nextcloud App Store{% block head-title %}{% endblock %}</title>
```

Comment:
Yeah, just for the record – of course also sub page titles should go first then. The most specific element should always go left. As I said in the OP:

> The more specific thing needs to go first so it works properly when you have a bunch of tabs open.